microsetta_interface/templates/survey.jinja2

```diff
@@ -163,7 +169,7 @@
                 let num_string_fields = document.getElementsByClassName("field-textArea").length + document.getElementsByClassName("field-input").length ;
                 if(form_submitted === 0) {
                     if(num_string_fields > 0 || Object.keys(survey_model).length>0) {
-                        confirm_submit = confirm("{{ _('Please note: Once you\'ve submitted this survey, you\'ll be unable to edit your answers.') }}\r\n\r\n{{ _('If you\'d like to review or change any of your responses, please press the Cancel button. Otherwise, press OK to submit your responses.') }}\r\n\r\n{{ _('This warning will only appear once.') }}");
```

Comment:
Should we retain the submission confirmation but instead update the message?

Reply:
I'm working on the front-end changes for the surveys and removing this warning.

Reply:
Thanks!

Reply:
Thanks @cassidysymons! I won't remove it from this PR then.